webapp/src/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
```

Comment:
:+1: A pain that this is required though

Reply:
You tell me. 😅